snuba/clickhouse/formatter.py

```diff
@@ -0,0 +1,95 @@
+from typing import Optional, Sequence
+from snuba.query.expressions import (
+    Column,
+    CurriedFunctionCall,
+    Expression,
+    ExpressionVisitor,
+    FunctionCall,
+    Literal,
+)
+from snuba.query.parsing import ParsingContext
+from snuba.util import escape_alias, escape_col, escape_string
+
+# Tokens used when formatting. Defining them as costant
```

Comment:
In the first implementation I was considering adding an abstraction for the Token. Looking at how simple the formatter actually is and knowing no further processing is supposed to happen on the formatted query, that seemed useless.

I think we could go back to this later if we wanted to support different ways of laying out the formatted query.